src/app/utilities/course/createCheckoutUrl.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import { createRequestConfig } from '../common';
+import { CourseCheckoutSession } from '@/app/interfaces/Course';
+import queryString from '../queryString';
+import { COURSE_TEST_ENROL_KEY } from './constants';
+
+const createCheckoutUrl = async (): Promise<CourseCheckoutSession> => {
+  return await axios
+    .request(
+      createRequestConfig(
+        '/course/createCheckoutUrl' +
+          queryString.stringify({
+            [COURSE_TEST_ENROL_KEY]: queryString.parse()[COURSE_TEST_ENROL_KEY],
```

Comment:
Do we intend to replace this by the actual enrolment key? why do we need to use test?